README.md

```diff
@@ -17,9 +17,9 @@ If you have a server-side Swift application, or maybe a cross-platform (e.g. Lin
 To add a dependency on the metrics API package, you need to declare it in your `Package.swift`:
 
 ```swift
-// SwiftMetrics is converging towards the 1.0.0 release as of April 22nd 2019, the tag is 1.0.0-convergence.1
+// as of May 5th 2019, SwiftMetrics major stable release is 1.0.0
```

Comment:
- "As of"
- "May 5, 2019" or "5 May 2019"

And I think it should be `SwiftMetrics' major stable release` (with apostrophe at the end) but maybe suffice to say "the major stable release".